crates/aptos/src/update/helpers.rs

```diff
@@ -61,4 +61,40 @@ impl InstallationMethod {
         };
         Ok(installation_method)
     }
+
+    pub fn get_instructions(&self) -> Option<&'static str> {
+        match self {
+            // Don't tell people to upgrade when they're building from source.
+            InstallationMethod::Source => None,
+            InstallationMethod::Homebrew => Some("brew upgrade aptos"),
+            InstallationMethod::Other => Some("aptos update"),
```

Comment:
You got `aptos update` running?

Reply:
Yeah in https://github.com/aptos-labs/aptos-core/pull/6288, so we'd have to land that first.